Pipfile

```diff
@@ -9,7 +9,7 @@ python_version = "3.6"
 [dev-packages]
 semantic-version = "2.8.5"
 setuptools-rust = "0.12.1"
-flake8 = "*"
+flake8 = "~=4.0.1"
```

Comment:
I had to lock flake8 to the 4.x version. Trying to run anything from 5.x or 6.x on my RHEL8 dev machine results in a max depth recursion error. I'm guessing it has something to do with python 3.6.